### DISCUSSION TITLE
[1.2.2] 2022-01-19

### DISCUSSION THREAD

**Discussion by darthcloud** [darthcloud/BlueRetro#148] [General]
# Bugfixes
* Fix regression from v1.2.1 that broken Wii extension support & PS5 rumble
* [BT] Detect SW ctrls stall and send sniff mode exit to unstall (Fixes #111)

<hr /><em>This discussion was created from the release <a href='https://github.com/darthcloud/BlueRetro/releases/tag/v1.2.2'>[1.2.2] 2022-01-19</a>.</em>

**Comment by Luizcsf** [comment]
Good evening! i installed blue retro on gamecube and it´s AWESOME!

I used the gc_spiffs.bin and when i put a wired controller on slot 2, it interferes with slot 1(even if the 1st controller is on bluetooth or wired too). 
Any suggestions?
The schematics are made exactly as planned.

In case of being a firmware issue, can i update it?

**Reply by darthcloud** [reply]
I suspect you made an internal mods and now to try using wired controller, and this is isn't supported yet.

It work for some by pure luck, and doesn't for many other. Nothing I can do, wait for official support.

**Reply by Luizcsf** [reply]
Yeah, i made an internal mod.

When i use only bluetooth controllers, it works wonders.
